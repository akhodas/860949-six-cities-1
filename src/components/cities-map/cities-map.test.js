import React from 'react';
import renderer from 'react-test-renderer';

import CitiesMap from './cities-map.jsx';
import mockLeaflet from '../../mocks/mock-leaflet';

describe(`CitiesMap`, () => {
  it(`correctly rendered`, () => {
    const tree = renderer
      .create(<CitiesMap
        offers={[]}
        leaflet={mockLeaflet}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
