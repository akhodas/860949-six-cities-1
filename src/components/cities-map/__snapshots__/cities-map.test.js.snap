// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesMap correctly rendered 1`] = `
<section
  className="cities__map map"
>
  <div
    id="map"
    style={
      Object {
        "height": "100%",
      }
    }
  />
</section>
`;
